app/app/Country.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Country extends Model
+{
+	protected $table = 'country';
+    protected $fillable = [
```

Comment:
This is pretty minor but maybe you can fix the inconsistent spacing between the protected $table = 'country'; line and protected $fillable = [.